event_webhook_types.go

```diff
@@ -426,28 +425,28 @@ type JobEvent struct {
 	ProjectName         string     `json:"project_name"`
 	User                *EventUser `json:"user"`
 	Commit              struct {
-		ID          int    `json:"id"`
-		Name        string `json:"name"`
-		SHA         string `json:"sha"`
-		Message     string `json:"message"`
-		AuthorName  string `json:"author_name"`
-		AuthorEmail string `json:"author_email"`
-		AuthorURL   string `json:"author_url"`
-		Status      string `json:"status"`
-		Duration    int    `json:"duration"`
-		StartedAt   string `json:"started_at"`
-		FinishedAt  string `json:"finished_at"`
+		ID          int     `json:"id"`
+		Name        string  `json:"name"`
+		SHA         string  `json:"sha"`
+		Message     string  `json:"message"`
+		AuthorName  string  `json:"author_name"`
+		AuthorEmail string  `json:"author_email"`
+		AuthorURL   string  `json:"author_url"`
+		Status      string  `json:"status"`
+		Duration    float64 `json:"duration"`
+		StartedAt   string  `json:"started_at"`
+		FinishedAt  string  `json:"finished_at"`
 	} `json:"commit"`
 	Repository *Repository `json:"repository"`
 	Runner     struct {
 		ID          int      `json:"id"`
 		Active      bool     `json:"active"`
 		RunnerType  string   `json:"runner_type"`
-		Shared      bool     `json:"is_shared"` // TODO variable name should be IsShared
```

Comment:
Does this break backwards compatibility? As in, was this field called `Shared` in older versions?

Reply:
Yes, it would break backward compatibility. It was called `Shared` in older versions